CHANGELOG.md

```diff
@@ -3,6 +3,64 @@
+* add a claimsRegistrar based on attestations ([#3622](https://github.com/Agoric/agoric-sdk/issues/3622)) ([3acf78d](https://github.com/Agoric/agoric-sdk/commit/3acf78d786fedbc2fe02792383ebcc2cadaa8db2)), closes [#3189](https://github.com/Agoric/agoric-sdk/issues/3189) [#3473](https://github.com/Agoric/agoric-sdk/issues/3473) [#3932](https://github.com/Agoric/agoric-sdk/issues/3932)
+* **react-wallet:** Integrate wallet connection component ([#3922](https://github.com/Agoric/agoric-sdk/issues/3922)) ([01a9118](https://github.com/Agoric/agoric-sdk/commit/01a91181e36f4e2dc49dcbb1327c50e3b268d2f9))
+* ContractGovernor manages parameter updating for a contract ([#3448](https://github.com/Agoric/agoric-sdk/issues/3448)) ([59ebde2](https://github.com/Agoric/agoric-sdk/commit/59ebde27708c0b3988f62a3626f9b092e148671f))
+* revise x/lien to hold total liened amount ([842c9b0](https://github.com/Agoric/agoric-sdk/commit/842c9b0d98a8655b286c9f91c70bb6fddc3e0ba3))
+* stateless lien module that upcalls to kernel ([603c0cf](https://github.com/Agoric/agoric-sdk/commit/603c0cfc8d2b4706dbbaa42d2ae057fa9dea65dc))
+* **wallet:** Create react ui scaffold with ses ([#3879](https://github.com/Agoric/agoric-sdk/issues/3879)) ([089c876](https://github.com/Agoric/agoric-sdk/commit/089c876d801efc1ede76b3011a1301384aace77f))
+* **wallet-backend:** add `walletAdmin.getClockNotifier()` ([2902ec3](https://github.com/Agoric/agoric-sdk/commit/2902ec38f6110a5582f6906695b77911f4d43180))
+* **wallet-backend:** attach timestamp and sequence metadata ([9e02962](https://github.com/Agoric/agoric-sdk/commit/9e02962b7c08d56ea1fe72970f3309998b734767))
+* **wallet-connection:** Add getAdminBootstrap functionality to wallet connection ([#3909](https://github.com/Agoric/agoric-sdk/issues/3909)) ([4c56367](https://github.com/Agoric/agoric-sdk/commit/4c563672836edaf92a65fb7829b0a189f7e4ce53))
+* **wallet-connection:** create new cross-framework UI ([905bf6c](https://github.com/Agoric/agoric-sdk/commit/905bf6ccef614b2b8b8d782810b252df9df1abcc))
+* **wallet-connection:** handle dapp approval states ([32b7772](https://github.com/Agoric/agoric-sdk/commit/32b7772ed33ed512ed598bbfc5dcea16ed36a705))
+* Thread URL and Base64 endowments ([b52269d](https://github.com/Agoric/agoric-sdk/commit/b52269d58be665baf45bbb38ace57ca741e5ae4c))
+
+
+### Bug Fixes
+
+* "yarn lint" should not fix ([#3943](https://github.com/Agoric/agoric-sdk/issues/3943)) ([0aca432](https://github.com/Agoric/agoric-sdk/commit/0aca432ed7a3f33eed4575b36dc3fa9e023b445f))
+* adapt timers to async iterables ([#3949](https://github.com/Agoric/agoric-sdk/issues/3949)) ([9739127](https://github.com/Agoric/agoric-sdk/commit/9739127262e9fac48757094a4d2d9f3f35f4bfc5))
+* address review comments ([8af3e15](https://github.com/Agoric/agoric-sdk/commit/8af3e1547b4df32c604f6b628a62bff230666166))
```

Comment:
"address review comments" is one of my least favorite commit headlines. Surely if the comments were worth addressing, there are two or three words to say what they were about. Or more likely: this wasn't really a fix to a bug that was ever on master, and it should have been squashed or tagged `chore` or `style` rather than `fix`.